pyramid_oereb/lib/renderer/__init__.py

```diff
@@ -143,3 +145,44 @@ def get_multilingual_text(self, values):
             list of dict: List of dictionaries containing the multilingual representation.
         """
         return [self.get_localized_text(values)]
+
+    @staticmethod
+    def unaccent_lower(text):
+        """
+        Replaces all special characters so that an alphabetical sorting can be done.
+
+        Args:
+            text (str): The text value.
+
+        Returns:
+            new_text (str): The text value converted to lower case and striped of special characters.
+        """
+        new_text = unicode(text.lower())
+        return unicodedata.normalize('NFD', new_text)
+
+    def sort_by_localized_text(self, element_list):
+        """
+        Sort a list of translated text elements alphabetically.
+
+        Args:
+            element_list (pyramid_oereb.lib.records.view_service): The list of map.legends to sort.
+
+        Returns:
+            element_list (pyramid_oereb.lib.records.view_service): Alphabetically and language
```

Comment:
Combine doc in in one section like: "Alphabetically and language specific sorted elements if translations exist or the list of unsorted map.legends if sorting was not possible."

add type like: "element_list (list of pyramid_oereb.lib.records.view_service)"